OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- mnairn
+- maleck13
+- philbrookes
+- sedroche
```

Comment:
I think you can remove `sedroche` (:cry:)

Reply:
:+1: